tests/dummy/app/controllers/logging-example/application-log.js

```diff
@@ -0,0 +1,375 @@
+import Ember from 'ember';
+import ListFormController from 'ember-flexberry/controllers/i-i-s-caseberry-logging-objects-application-log-l';
+const { getOwner } = Ember;
+
+export default ListFormController.extend({
+  /**
+  Flexberry Logging Service
+
+   @property flexberryLoggingService
+   @type String[]
+   */
+  flexberryLoggingService: Ember.inject.service('flexberry-logging'),
+
+  _messageNumber: 0,
+
+  logLevel: 0,
+
+  /**
+    Available test application level settings
+
+    @property settings
+    @type String[]
+   */
+  settings: [],
+
+  /**
+   Default choise in settings
+
+   @property text
+   @type String[]
+   */
+  text: '',
+
+  /**
+   List buttons describers for Supported actions
+   */
+  customButtons:  [{
+      buttonName: 'Assert',
```

Comment:
We make localized version of stand.
There is an example of custom buttons localization on stand.